### DISCUSSION TITLE
How to remove scenes where the file no longer exists?

### DISCUSSION THREAD

**Discussion by ghost** [stashapp/stash#2787] [Q&A] [answered]
I'm using [this script](https://lemmy.ml/post/382850) to create a virtual file system from my stash database using [Supertag](https://amoffat.github.io/supertag/) but it's telling me there are many scenes for which the file doesn't exist. How can I remove all those scenes from the database?

**Comment by MrX292** [comment, marked as answer]
under settings > Tasks > Maintenance > Clean.  i would suggest running it in dry mode first to see what it will clean

**Comment by losthesodium** [comment]
FYI to fix this issue I've started deleting from the web interface, just select the checkbox for delete file.

**Comment by WithoutPants** [comment]
@JediMaster25 if you edit discussions to remove the title/content again, you will be blocked from the repository. Do not do this again.
